src/main/java/com/marginallyclever/communications/NetworkSession.java

```diff
@@ -30,6 +31,8 @@ public String getName() {
 	
 	// OBSERVER PATTERN
 
+	private List<NetworkSessionListener> listeners = new ArrayList<>();
```

Comment:
Why is it better?

Reply:
I moved it to make it homogen as other classes. I like this organization with the comment: `// OBSERVER PATTERN`: it makes it clear.